proj/src/reducers/time-tracking/timeTrackingReducer.js

```diff
@@ -0,0 +1,24 @@
+import { handleActions, combineActions } from 'redux-actions';
+import { set } from 'lodash/fp'
+
+const initialState = {
+  offset: 0,
+  dates: [],
+  hours: {},
+  displayHours: false
+};
+
+export default handleActions({
+  SET_DATES: (state, { dates }) => 
+    set('dates', dates, state )
+  ,
+  UPDATE_OFFSET: (state, { offset }) => {
+    if(offset === 7) {
```

Comment:
i would prefer having this logic in a utils function and probably in the component so the reducer has less logic in it